app/templates/homepage.html

```diff
@@ -98,13 +98,19 @@ <h2 class="contact__heading">Contact Us</h2>
         <input type="submit" value="Send" class="button button-blue button-block">
       </form>
     </div>
-
     <form class="section--contact__paypal" action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
-    <input type="hidden" name="cmd" value="_s-xclick">
-    <input type="hidden" name="hosted_button_id" value="PZNFQVFPK8RFG">
-    <input type="image" src="https://www.paypalobjects.com/en_US/i/btn/btn_donate_LG.gif" border="0" name="submit" alt="PayPal - The safer, easier way to pay online!">
-    <img alt="" border="0" src="https://www.paypalobjects.com/en_US/i/scr/pixel.gif" width="1" height="1">
+      <input type="hidden" name="cmd" value="_s-xclick">
+      <input type="hidden" name="hosted_button_id" value="PZNFQVFPK8RFG">
```

Comment:
I would add the value here from a config file with angular. I wouldn't hardcode this here. but It might be ok for now